src/components/Pagination.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import classNames from 'classnames';
+
+interface Props {
+  total: number,
+  perPage: number,
+  page: number,
+  handleClick: any,
+}
+
+const defaultProps = {
+  withInfo: false,
+};
+
+const Pagination: React.FC<Props & typeof defaultProps> = ({
+  total,
+  perPage,
+  page,
+  withInfo,
+  handleClick,
+}) => {
+  const totalPages = Math.ceil(total / perPage);
+  const startItem = perPage * (page - 1) + 1;
+  const endItem = perPage * page > total ? total : perPage * page;
```

Comment:
```suggestion
  const endItem = perPage * page > total 
    ? total 
    : perPage * page;
```
а ще краще створити змінну для perPage * page